### ISSUE TITLE
add support for Soumi music player

### ISSUE THREAD

**Issue by GraefinZahl** [tgwizard/sls#293]
It's the best smb network music player for me. Is there a possibility to add support?

**Comment by HumbleBeeBumbleBee** [comment]
https://play.google.com/store/apps/details?id=com.techsoftpl.musicplayer ?

**Comment by HumbleBeeBumbleBee** [comment]
I have sent an email to support@techsoftpl.com requesting they broadcast song information so SLS can scrobble music played in Soumi.

**Comment by GraefinZahl** [comment]
Yes, this is the correct player, thanks for taking a look at it! :) 👍
